### ISSUE TITLE
A native Apple Silicon version

### ISSUE THREAD

**Issue by gleek12** [tahoma2d/tahoma2d#1444]
Apologies if this request already exists somewhere, I couldn't find one.

It would be amazing if we could get Tohoma3D/Opentoonz ported over to the Apple Silicon chips.   It runs decently in Rosetta for now, but a native version would likely be far more performant.  Additionally, Apple has a history of deprecating legacy software after a few years, making me think that non-native apps are running on borrowed time, and it would be horrible if Opentoonz/Tahoma2D just simply stopped running on new versions of MacOS.

**Comment by artkiver** [comment]
> I believe once OT is updated to QT6, T2D will follow. Once that happens we should be able to have builds for both Intel and Apple Silicon. It might not be really soon, since it'll be a time consuming upgrade. https://doc.qt.io/qt-6/macos.html#architectures

I am pretty sure that aarch64/arm64/ARM/Apple Silicon is supported even on QT5 from version 5.15, also see:

https://doc.qt.io/qt-5/macos.html#architectures

(additional discourse here: https://forum.qt.io/topic/139777/qt-5-15-10-and-apple-silicon-m1-m2-support/4)

Or look at the Port Health in Qt5 versions in MacPorts, e.g. https://ports.macports.org/port/qt5/details/ which shows qt5.15.12 building successfully on "arm64" (aka Apple Silicon) and x86_64 (aka legacy Intel) on macOS Sonoma, Ventura, Monterey and Big Sur (the first macOS release with Apple Silicon hardware).

TL;DR, not to dissuade the Tahoma2D developers from embracing Qt6, but according to upstream documentation (as well as knowing other projects which are using Qt5's libraries on Apple Silicon natively) I think that lack of Qt6 adoption probably isn't the most likely roadblocK when it comes to building on Apple Silicon?

**Comment by artisteacher** [comment]
Well the biggest roadblock is not currently having a developer with an apple silicon device to develop with...
